### PR TITLE
IDEX-3284: Show all Hotkey that available in Che 

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
Show all Hotkey that available in Che (including keys for Orion)

### CODE REVIEW

File: plugin-java/che-plugin-java-ext-lang-client/src/main/resources/org/eclipse/che/ide/ext/java/client/JavaLocalizationConstant.properties [hunk 1]
```diff
@@ -50,6 +50,7 @@ move.preview.title = Changes to be performed
 multi.selection.destination={0} selected elements:
 multi.selection.references={0} moved elements
 rename.refactoring.action.name=Rename
+rename.refactoring.action.description=Rename java file refactoring
```
Comment (sleshchenko) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/resources/org/eclipse/che/ide/ext/java/client/JavaLocalizationConstant.properties]: I think it is wrong description

Comment (evoevodin) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/resources/org/eclipse/che/ide/ext/java/client/JavaLocalizationConstant.properties]: I would say it is a bit verbose :smile:

File: plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/jso/OrionKeyBindings_Overlay.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.editor.orion.client.jso;
+
+import com.google.gwt.core.client.JavaScriptObject;
+
+/**
+ * Class that bind actionId and keyBindings for this action
+ */
+public class OrionKeyBindings_Overlay extends JavaScriptObject {
```
Comment (vparfonov) [inline plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/jso/OrionKeyBindings_Overlay.java]: Why you use "_" in class name ?

**Comment by AndrienkoAleksandr** [comment]
merged with improvements https://github.com/codenvy/che-plugins/pull/401

**Pull request closed without merging**